english/views.py

```diff
@@ -16,7 +17,7 @@ class ClassListAPIView(ListAPIView):
 class CurrentClassesForStudentListAPIView(ListAPIView):
     queryset = Class.objects.all()
     serializer_class = ClassSerializer
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsEmployee]
```

Comment:
It is better to use IsAuthenticated, IsEmployee

Reply:
`IsEmployee` already has validators for authorization

Reply:
What status returns IsEmployee? And IsAuthenticated?